ocaml/xapi/xapi_host.ml

```diff
@@ -1103,8 +1103,12 @@ let syslog_reconfigure ~__context ~host:_ =
   let flag =
     match destination with "" -> "--noremote" | _ -> "--remote=" ^ destination
   in
-  let args = [|!Xapi_globs.xe_syslog_reconfigure; flag|] in
-  ignore (Unixext.spawnvp args.(0) args)
+  let _ =
```

Comment:
Usually it is good to specify the type:
```
let (_: string*string) = ...
```

That way if forkhelpers.execute_command_get_output gains more parameters this won't silently stop working, although in its current form it is correct.